scripts/bootloader/do_sign.py

```diff
@@ -31,9 +32,9 @@ def parse_args():
     return args
 
 
-if __name__ == '__main__':
+if __name__ == 'main':
     args = parse_args()
-    private_key = SigningKey.from_pem(args.private_key.read())
+        private_key = SigningKey.from_pem(args.private_key.read())
     data = args.infile.read()
     signature = private_key.sign(data, hashfunc=hashlib.sha256)
     args.outfile.write(signature)
```

Comment:
- The `if __name__ == 'main':` check should be `if __name__ == '__main__':`.  
```suggestion
if __name__ == '__main__':
```
- There is incorrect indentation for the line `private_key = SigningKey.from_pem(args.private_key.read())`.  
```suggestion
    private_key = SigningKey.from_pem(args.private_key.read())
```